mlflow/ml-package-versions.yml

```diff
@@ -2,8 +2,7 @@ sklearn:
   package_info:
     pip_release: "scikit-learn"
     install_dev: |
-      head_sha=$(git ls-remote https://github.com/scikit-learn/scikit-learn.git HEAD | cut -f1)
-      pip install git+https://github.com/scikit-learn/scikit-learn.git@$head_sha
+      pip install git+https://github.com/scikit-learn/scikit-learn.git
```

Comment:
I added `head_sha` to make it easier to identify the commit SHA but `pip freeze` shows it:

https://github.com/mlflow/mlflow/pull/4534/checks?check_run_id=2999114540#step:10:67

```
scikit-learn @ git+https://github.com/scikit-learn/scikit-learn.git@f2fb07581339f063b501b3e820917d66ee4c1fa3
```